changelog.d/20241028_113440_jonathan.griffe_fix_precommit_git_merge_filename_with_spaces.md

```diff
@@ -0,0 +1,3 @@
+### Fixed
+
+- The ggshield pre-commit hook no longer crashes when merging files with spaces in their names.
```

Comment:
This fixes https://github.com/GitGuardian/ggshield/issues/991, right?

If so, it's a good thing to add it here, this way a link to the issue will be created in the release notes:

```suggestion
- The ggshield pre-commit hook no longer crashes when merging files with spaces in their names (#991).
```

Reply:
It does ! Thank you for the comment I will change it